tools/deb/BUILD

```diff
@@ -0,0 +1,51 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    srcs = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
```

Comment:
Agreed it shouldn't be /usr/local - but separate PRs needed across pilot.

On /opt - I know it's controversial, but seems to be ok for google-chrome. I'll check what k8s does. 

We can fine tune this - but we need to get testing started.